src/book/03-guides/04-general/03-troubleshooting.mdx

```diff
@@ -11,16 +11,7 @@ authors:
 
 ## General
 
-If this page does not solve your problem, the following people can help you with specific parts of the system:
-
-- **Hardware**: Taylor Young
-- **Toolchain and build**: Kip Hamiltons
-- **NUClear**: Cameron Murtagh
-- **NUsight**: Josephus Paye II, Brendan Annable
-- **Machine Learning**: Joel Wong
-- **Mathematics**: Kip Hamiltons, Cameron Murtagh, Taylor Young, Ysobel Sims
-
-Team mentors **Alex Biddulph** and **Trent Houliston** can help with most parts of the system if this list does not help.
+If this page does not solve your problem, ask for help on Slack either in a channel or directly to a senior team member.
```

Comment:
```suggestion
If this page does not solve your problem, ask for help on Slack either in a channel by directly messaging a senior team member.
```

Reply:
I think you need to reread what you put :stuck_out_tongue: 
Maybe
```suggestion
If this page does not solve your problem, ask for help on Slack by messaging a channel or messaging a senior team member directly.
```

Reply:
@LachlanCourt  ^^^ This suggestion changes the meaning of the sentence though right? I read the original sentence as "ask in Slack" or "ask a senior member directly". The latter could be in person etc.